app/routes/events.tsx

```diff
@@ -0,0 +1,31 @@
+import { useLoaderData } from "@remix-run/react";
+import { MetaFunction, json } from "@vercel/remix";
+
+import { EventsList } from "~/components/EventsList";
```

Comment:
(What is this `~` when importing components? I have never seen this)

Reply:
Oh that's really neat, I've never actually used that, and I'm going to change that! :D